abis/FairSale.json

```diff
@@ -159,7 +159,7 @@
       {
         "indexed": false,
         "internalType": "uint96",
-        "name": "auctionedTokens",
+        "name": "saleedTokens",
```

Comment:
How did you update this ABI? Mesa contracts do not have the varaible `saleedTokens`. ` Get them from the artifacts and run `build-abis.sh` then update typechain types.